vue-app/src/api/recipient-registry-kleros.ts

```diff
@@ -79,19 +79,19 @@ export async function getProjects(
   const projects: Project[] = []
   for (const event of recipientAddedEvents) {
     const project = decodeRecipientAdded(event, tcrColumns)
-    if (endBlock && event.blockNumber >= endBlock) {
+    const addedAt = (event.args as any)._timestamp.toNumber()
+    if (endTime && addedAt >= endTime) {
       // Hide recipient if it is added after the end of round.
-      // We can not do this with filter because on xDai node returns
-      // "One of the blocks specified in filter ... cannot be found"
       project.isHidden = true
     }
     const removed = recipientRemovedEvents.find((event) => {
       return (event.args as any)._tcrItemId === project.id
     })
     if (removed) {
-      if (!startBlock || startBlock && removed.blockNumber <= startBlock) {
-        // Start block not specified
-        // or recipient had been removed before start block
+      const removedAt = (removed.args as any)._timestamp.toNumber()
+      if (!startTime || startTime && removedAt <= startTime) {
```

Comment:
The second `startTime` is logically redundant. Do you think it's more clear or less clear than this simplification?
```suggestion
      if (!startTime || removedAt <= startTime) {
```

Reply:
I agree, it should be simplified.